dockers/docker-fpm-frr/bgpcfgd

```diff
@@ -1,64 +1,117 @@
 #!/usr/bin/env python
 
 import sys
+import copy
```

Comment:
Please provide in your PR a full and detailed description of what you are trying to achieve.